### ISSUE TITLE
onlyAllowTags doesn't support case insensitive tags

### ISSUE THREAD

**Issue by Rhababo** [JiLiZART/BBob#190]
The allowed tags settings doesn't have an option for checking for case sensitivity. 

One option to fix this is to add a .toLowerCase() to the isAllowedTag() function in the parser

```
const isAllowedTag = (value) => {
    if (options.onlyAllowTags && options.onlyAllowTags.length) {
      return options.onlyAllowTags.indexOf(value**.toLowerCase()**) >= 0;
    }

    return true;
  };
```
toTagNode() already contains a .toLowerCase() statement

```
toTagNode() {
return new TagNode(this.tag.toLowerCase(), this.attrs, this.content);
}

```
so all tags get converted to lowercase anyway.

Could we either add .toLowerCase() to the isAllowedTag function, or, if we want block incorrectly cased tags, can we add an option that when flagged allows for case insensitive tags?

**Comment by JiLiZART** [comment]
bbcode like an a html ignoring case of tags, so i think parser and `onlyAllowTags` also should ignore the case of tags. I fix this behavior in next release.

**Comment by Mondrethos** [comment]
> bbcode like an a html is ignoring case of tags, so i think the parser and`onlyAllowTags` also should ignore the case of tags. I fix this behavior in next release.

Thanks! Pretty excited for this fix.

**Comment by JiLiZART** [comment]
Fixed in 3.0.2
